src/main/java/werkit/WerkIt.java

```diff
@@ -167,48 +167,77 @@ private void loadRequiredDirectoryAndFiles() throws IOException {
      * @throws IOException If the application is unable to process the user input.
      */
     public void startContinuousUserPrompt() throws IOException {
-        boolean userWantsToExit = false;
         boolean isFirstPrompt = true;
 
+        Command newCommand;
         do {
-            try {
-                getUI().printUserInputPrompt(isFirstPrompt);
-                isFirstPrompt = false;
-                String userInput = getUI().getUserInput();
-                Command newCommand = getParser().parseUserInput(userInput);
-
-                if (newCommand instanceof ExitCommand) {
-                    userWantsToExit = true;
-                    continue;
-                }
-
-                newCommand.execute();
-                if (newCommand instanceof WorkoutCommand) {
-                    if (newCommand.getUserAction().equals(DELETE_ACTION_KEYWORD)) {
-                        reloadScheduleFile();
-                    }
-                }   else if (newCommand instanceof PlanCommand) {
-                    if (newCommand.getUserAction().equals(DELETE_ACTION_KEYWORD)) {
-                        reloadScheduleFile();
```

Comment:
seems like we have a clash in refactoring code here, I will discard my changes here

Reply:
Oh, okay. Thanks :) Sorry for the inconvenience!